sys-fs/extundelete/files/extundelete-0.2.4-clang.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/extundelete.cc
++++ b/src/extundelete.cc
+@@ -1268,7 +1268,7 @@
+ 	*new_ino = 0;
+ 	priv->ret_ino = new_ino;
+ 	priv->curr_name = curr_part;
+-	struct dir_context ctx = {search_flags, DIRENT_FLAG_INCLUDE_REMOVED,
++	struct dir_context ctx = {static_cast<unsigned int>(search_flags), DIRENT_FLAG_INCLUDE_REMOVED,
```

Comment:
Actually, let's go for the version in https://bugs.gentoo.org/732284.

Please also use Closes: tags in commits.

Thank you!